services/wfm/src/main/java/org/openkilda/wfm/topology/nbworker/bolts/LinkOperationsBolt.java

```diff
@@ -112,6 +117,41 @@ public LinkOperationsBolt(Auth neoAuth) {
         return results;
     }
 
+    private List<FlowsResponse> getFlowIdsForLink(GetFlowIdsForLinkRequest request, Session session) {
```

Comment:
Perhaps it makes sense to place such method in FlowOperationsBolt

Reply:
+1 As this is related to flows.

Reply:
Moved it to FlowOperationsBolt.